security_monkey/alerters/slack_alerter.py

```diff
@@ -0,0 +1,44 @@
+"""alert slack with any auditor"""
+
+import json
+
+import requests
+from security_monkey import app
+from security_monkey.alerters import custom_alerter
+
+
+class SlackAlerter(object):
+    __metaclass__ = custom_alerter.AlerterType
+
+    def __init__(self, cls, name, bases, attrs):
+        super().__init__(cls, name, bases, attrs)
+        self.slack_config = {}
+        self.slack_hook = ""
+
+        if app.config.get('SLACK_HOOK'):
+
+            self.slack_config = {
+                'channel': app.config.get('SLACK_CHANNEL'),
+                'username': app.config.get('SLACK_USERNAME'),
+                'icon_emoji': app.config.get('SLACK_ICON'),
+            }
+
+            self.slack_hook = app.config.get('SLACK_HOOK')
+
+            self.slack_config['attachments'] = []
+
+    def report_auditor_changes(self, auditor):
+        for item in auditor.items:
+            for issue in item.confirmed_new_issues:
+                x = {
+                    'text': "New Issue: Index: {!s}\n Account: {!s}\n Region: {!s}\n Name: {!s}".format(item.index, item.account, item.region, item.name)
+                }
+
+                self.slack_config['attachments'].append(x)
```

Comment:
Make `attachments` a local variable, and then this should be good to go 👍

Reply:
attachments needs to be part of slack_config to be posted correctly into the slack hook. I set it equal to the message, and removed it from the init method.

Reply:
Makes sense.